src/Root.cpp

```diff
@@ -192,6 +194,8 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 	// cClientHandle::FASTBREAK_PERCENTAGE = settingsRepo->GetValueSetI("AntiCheat", "FastBreakPercentage", 97) / 100.0f;
 	cClientHandle::FASTBREAK_PERCENTAGE = 0;  // AntiCheat disabled due to bugs. We will enabled it once they are fixed. See #3506.
 
+	LoadPalettes(settingsRepo->GetValueSet("Folders", "ProtocolPalettes", "Protocol"));
```

Comment:
By loading the path from settings.ini, it is possible for developers to overcome the slow-load of multiple palettes in debug mode (#4453) by specifying a folder with just one protocol version instead of the default folder.